skops/io/_scipy.py

```diff
@@ -11,7 +12,7 @@ def sparse_matrix_get_state(obj, dst):
         "__module__": get_module(type(obj)),
     }
 
-    f_name = f"{uuid4()}.npz"
+    f_name = Path(obj) / f"{uuid4()}.npz"
```

Comment:
could you push this one directly to the repo please?

Also, wondering why this wasn't caught by tests.